CHANGELOG-1.0.md

```diff
@@ -0,0 +1,120 @@
+# Release notes for 1.0.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 0.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Adding configurable QPS and Burst to k8s client and use a separate client for leader election. ([#91](https://github.com/kubernetes-csi/external-resizer/pull/91), [@RaunakShah](https://github.com/RaunakShah))
```

Comment:
are these new args documented? you can see the message i did for provisioner: https://github.com/kubernetes-csi/external-provisioner/pull/459

Reply:
they are now.